### PR TITLE
Add 'from address' option in transfer dialog advanced part

### PR DESCRIPTION
**Pull request by Lichen9618**
Add feature according to #283

### CODE REVIEW

File: neo-gui/UI/TransferDialog.cs [hunk 1]
```diff
@@ -42,13 +44,31 @@ public Transaction GetTransaction()
...
             {
                 tx = new ContractTransaction();
             }
             else
             {
-                UInt160[] addresses = Program.CurrentWallet.GetAccounts().Select(p => p.ScriptHash).ToArray();
+                UInt160[] addresses;
+                if (FromAddress != null)
+                {
+                    addresses = (from add in Program.CurrentWallet.GetAccounts().Select(p => p.ScriptHash).ToArray() where add.Equals(FromAddress) select add).ToArray();
```
Comment (shargon) [inline neo-gui/UI/TransferDialog.cs]: We usually use the other syntax type
`.Where(u=>u...)`

Comment (Lichen9618) [reply neo-gui/UI/TransferDialog.cs]: @shargon 
Understand, I will modify them soon.

Thanks

**Review by shargon — APPROVED** [review]

File: neo-gui/UI/TransferDialog.Designer.cs [hunk 2]
```diff
@@ -36,6 +36,8 @@ private void InitializeComponent()
             this.button3 = new System.Windows.Forms.Button();
             this.button2 = new System.Windows.Forms.Button();
             this.groupBox1 = new System.Windows.Forms.GroupBox();
+            this.comboBox2 = new System.Windows.Forms.ComboBox();
+            this.label3 = new System.Windows.Forms.Label();
```
Comment (lock9) [inline neo-gui/UI/TransferDialog.Designer.cs:40]: Maybe next time we could use more clear names for these components. It is hard to review the code with these names.

**Review by lock9 — COMMENTED** [review]
It seems to be working, but I didn't test with real assets.
@shargon  did you do that?
Not sure if this should be merged to master, or dev branch instead (until proper Q.A. is done).

File: neo-gui/UI/TransferDialog.Designer.cs (hunk 2 above)
Comment (Lichen9618) [reply neo-gui/UI/TransferDialog.Designer.cs:40]: @lock9 It have been tested about UTXO and NEP5 asset by NGD test OP. Function works all right. 

About name, I agree we should take better name. In next version, I will try to do that.

**Pull request merged**